apps/prairielearn/assets/scripts/instructorAiGenerateDraftsSampleQuestions.tsx

```
@@ -0,0 +1,193 @@
+
+                            <p class="fw-bold mb-1 mt-3">Prompt</p>
+                            <p id="sample-question-prompt">{examplePrompts[0].promptGeneral}</p>
+                            <button id="fill-prompts" type="button" class="btn btn-primary me-2">
+                                <i class="fa fa-clone mr-2" aria-hidden="true"></i>
+                                Fill prompt
+                            </button>
+                        </div>
+                    </div>
+                </div>
+            </div>  
+        </div>
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)